rust_snuba/rust_arroyo/Cargo.lock

```diff
@@ -0,0 +1,1817 @@
+# This file is automatically @generated by Cargo.
```

Comment:
we should not check in this file, only for rust_snuba

rust_arroyo is a library, rust_snuba is an application